.github/renovate.json

```
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended"
+    ],
+    "configMigration": true,
+    "lockFileMaintenance": {
+        "enabled": true,
+        "automerge": true,
+        "automergeType": "branch",
+        "ignoreTests": true
+    },
+    "schedule": [
+        "before 4am on the first day of the month"
```

I think I prefer once weekly. This notation would work, I assume?

```suggestion
        "before 4am on tuesday"
```

---

i think so, let me check real quick, there's a validator for the config
https://docs.renovatebot.com/config-validation/